src/static/index.html

```diff
@@ -0,0 +1,324 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="description" content="">
+    <meta name="author" content="">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <div class="container">
+        <div class="title">Repo Report!</div>
+        <div class="instructions">
+            <div>Enter Command and press Enter!</div>
+            <div id='status' class="status-hide">
+                <div class="loader">
+                    <div></div>
+                    <div></div>
+                    <div></div>
+                    <div></div>
+                </div>
+                <div class="status-text">Getting Data from Github...</div>
+            </div>
+        </div>
+
+        <input class="command" type="text" name="command" id="command">
+        <div class="table-container">
+            <table class="table" cellspacing="0" cellpadding="0">
+                <thead id="table-head" class="table-head">
+                </thead>
+                <tbody id='table-body'>
+                </tbody>
+            </table>
+        </div>
+    </div>
+    <style>
```

Comment:
should/can this be a separate .css file?

Reply:
Done.